### PR TITLE
chore: Migrate from TSLint to ESLint

### PR DESCRIPTION
**Pull request by code0100fun**

### CODE REVIEW

File: examples/s3-update-example-cli/package.json [hunk 1]
```diff
@@ -75,10 +75,9 @@
   "private": true,
   "repository": "jdxcode/s3-update-example-cli",
   "scripts": {
-    "postpack": "rm -f oclif.manifest.json",
-    "posttest": "tsc -p test --noEmit && tslint -p test -t stylish",
```
Comment (RasPhilCo) [inline examples/s3-update-example-cli/package.json]: `posttest` got dropped

Comment (RasPhilCo) [inline examples/s3-update-example-cli/package.json]: `posttest` got dropped

File: src/commands/update.ts [hunk 2]
```diff
@@ -251,7 +257,7 @@ get_script_dir () {
   echo "$DIR"
 }
 DIR=$(get_script_dir)
-${binPathEnvVar}="\$DIR/${bin}" ${redirectedEnvVar}=1 "$DIR/../${version}/bin/${bin}" "$@"
```
Comment (RasPhilCo) [inline src/commands/update.ts]: concerned about this one too

File: examples/s3-update-example-cli/package.json (hunk 1 above)
Comment (code0100fun) [reply examples/s3-update-example-cli/package.json]: It's still there. I think it was below this line somewhere

File: src/commands/update.ts (hunk 2 above)
Comment (code0100fun) [reply src/commands/update.ts]: Ok, I added this back.

**Pull request merged**